### PR TITLE
Add grafana package.

### PR DESCRIPTION
**Pull request by dlorenc**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [X] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

**Review by imjasonh — DISMISSED** [review]

File: grafana.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/grafana/grafana
+      tag: v${{package.version}}
+      expected-commit: ae830f687450b8a0aca94ab2d72cc08853a80fff
+
+  - runs: |
+      # Mititage CVE-2022-41723 / GHSA-vvpx-j8f3-3w6h
```
Comment (vaikas) [inline grafana.yaml:35]: nit: mitigate, and below

File: grafana.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      go mod edit -replace golang.org/x/net=golang.org/x/net@v0.17.0
+      go mod edit -replace go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp=go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp@v0.44.0
+      go get go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace@v0.44.0
+      go mod tidy
+
+      yarn install
+      yarn run build
+      yarn run plugins:build-bundled
+
+      # Our default LDFLAGS cause an issue with grafana. We need to remove '-z, noexecheap' from LDFLAGS
```
Comment (vaikas) [inline grafana.yaml:46]: should we create an issue tracking what the issue is?

Comment (vaikas) [reply grafana.yaml:46]: or document what the issue is so if later it gets fixed we can remove this.

Comment (kaniini) [reply grafana.yaml:46]: What issue does it cause?  In general, programs should not crash with this flag, as they should not be executing code from memory that is on the heap.

Comment (dlorenc) [reply grafana.yaml:46]: Filed an issue: https://github.com/wolfi-dev/os/issues/7419

**Review by rawlingsj — DISMISSED** [review]

File: grafana.yaml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    - license: AGPL-3.0-or-later
+  dependencies:
+    runtime:
+      - bash
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
```
Comment (hectorj2f) [inline grafana.yaml:16]: nit: Generally `ca-certificates-bundle` is included in `build-base --> wolfi-baselayout.`

**Review by hectorj2f — APPROVED** [review]

**Pull request merged**